### PR TITLE
created team linkboxes

### PR DESCRIPTION
**Pull request by Urs-sein-github**
Got everything running,
Created linkboxes,
was funny.

### CODE REVIEW

File: combiner/npm-debug.log [hunk 1]
```diff
@@ -0,0 +1,46 @@
+0 info it worked if it ends with ok
```
Comment (TomasKostadinov) [inline combiner/npm-debug.log]: Please delete this file

File: npm-debug.log [hunk 2]
```diff
@@ -0,0 +1,46 @@
+0 info it worked if it ends with ok
```
Comment (TomasKostadinov) [inline npm-debug.log]: Delete this file

File: site/team/abteilungsleitung/index.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    <div class="container">
+        <!-- SIDEBAR vorlage -->
+        <h1>Sidebar</h1>
+        <div class="main-content">
+            <div class="sidebar">
+                <div class="sticky">
+                    <div class="description">
+                        <h3>Auf dieser Seite</h3>
+                    </div>
+
```
Comment (TomasKostadinov) [inline site/team/abteilungsleitung/index.html]: Sidebar is not needed on this page

File: site/team/laufen/index.html [hunk 4]
```diff
@@ -0,0 +1,37 @@
+<div class="row">
+        <div class="container">
+        </br>
+        </br>
+        </br>
+        </br>
```
Comment (TomasKostadinov) [inline site/team/laufen/index.html]: ?

This does not change anything, remove the /br tags.


To add some space you can change the container to "container distance"

File: site/team/laufen/trainer_laufen/index.html [hunk 5]
```diff
@@ -0,0 +1,37 @@
+<div class="row">
+        <div class="container">
+        </br>
+        </br>
+        </br>
+        </br>
```
Comment (TomasKostadinov) [inline site/team/laufen/trainer_laufen/index.html]: Same here

File: site/team/laufen/trainer_laufen/trainer2/index.html [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+        
+        <img src="/assets/images/team_pics/Laufen/2019KarinWagner_lowspec.jpg">
+        
+        
+        <p><strong>Funktion: </strong>Betreuerin von Lauf- und Walkinggruppen</p>
+        <p><strong>Name: </strong>Karin Wagner</p>
+        <p><strong>Mein Alter</strong>: AK60</p>
+        <p><strong>Meine liebste Sportart</strong>: -</p>
+        <p><strong>Meine liebste Trainingsform</strong></p>
+        <p><strong>Mein h&auml;rtester Wettkampf</strong>: -</p>
```
Comment (TomasKostadinov) [inline site/team/laufen/trainer_laufen/trainer2/index.html]: can you decide wheter this shuld be a list or all other pages should not be a list so we dont have multiple layouts

File: site/team/leichtathletik/index.html [hunk 7]
```diff
@@ -0,0 +1,37 @@
+<div class="row">
+        <div class="container">
+        </br>
+        </br>
+        </br>
+        </br>
+        </br>
```
Comment (TomasKostadinov) [inline site/team/leichtathletik/index.html]: Change this

File: site/team/leichtathletik/spartenleitung_leichtathletik/index.html [hunk 8]
```diff
@@ -0,0 +1,26 @@
+<div class="container distance">
+</br>
+
```
Comment (TomasKostadinov) [inline site/team/leichtathletik/spartenleitung_leichtathletik/index.html:3]: remove this

File: site/team/leichtathletik/spartenleitung_leichtathletik/index.html [hunk 9]
```diff
@@ -0,0 +1,26 @@
+<div class="container distance">
+</br>
+    
+    <h2>Hey, ich bin der Steffen Heller</h2>
+    
+    <img src="">
+
```
Comment (TomasKostadinov) [inline site/team/leichtathletik/spartenleitung_leichtathletik/index.html:7]: No image source so we dont need this tag

File: site/team/leichtathletik/trainer_leichtathletik/index.html [hunk 10]
```diff
@@ -0,0 +1,15 @@
+<meta http-equiv="refresh" content="5; url=https://www.fetzer-bau.de/" />
+<div class="row">
+    <div class="container">
+    </br>
+</br>
+</br>
+</br>
+</br>
```
Comment (TomasKostadinov) [inline site/team/leichtathletik/trainer_leichtathletik/index.html]: Remove this

File: site/team/leichtathletik/trainer_leichtathletik/index.html [hunk 11]
```diff
@@ -0,0 +1,15 @@
+<meta http-equiv="refresh" content="5; url=https://www.fetzer-bau.de/" />
```
Comment (TomasKostadinov) [inline site/team/leichtathletik/trainer_leichtathletik/index.html:1]: 🙄😅

File: site/team/organisation_silvesterlauf/index.html [hunk 12]
```diff
@@ -0,0 +1,49 @@
+<div class="row">
+        <div class="container">
+        </br>
+        </br>
+        </br>
+        </br>
+        </br>
```
Comment (TomasKostadinov) [inline site/team/organisation_silvesterlauf/index.html]: Remove this

File: site/team/presse/index.html [hunk 13]
```diff
@@ -0,0 +1,15 @@
+<meta http-equiv="refresh" content="5; url=https://www.fetzer-bau.de/" />
+<div class="row">
+    <div class="container">
+    </br>
+</br>
+</br>
+</br>
+</br>
```
Comment (TomasKostadinov) [inline site/team/presse/index.html]: Remove this

File: site/team/presse/index.html [hunk 14]
```diff
@@ -0,0 +1,15 @@
+<meta http-equiv="refresh" content="5; url=https://www.fetzer-bau.de/" />
```
Comment (TomasKostadinov) [inline site/team/presse/index.html]: ...

File: site/team/triathlon/index.html [hunk 15]
```diff
@@ -0,0 +1,37 @@
+<div class="row">
+        <div class="container">
+        </br>
+        </br>
+        </br>
+        </br>
```
Comment (TomasKostadinov) [inline site/team/triathlon/index.html]: Change this

File: site/team/triathlon/spartenleitung_triathlon/index.html [hunk 16]
```diff
@@ -0,0 +1,14 @@
...
+</br>
+    
+    <h2>Hey, ich bin der Marcus Pfeffner</h2>
+    
+    <img src="/assets/images/team_pics/Triathlon/PfefferMarcus_lowspec.jpg">
+    
+    <p>&nbsp;</p>
+    <p>Team</p>
+    <p>Funktion: Spartenleiter Triathlon</p>
+    <p>Name: Marcus</p>
```
Comment (TomasKostadinov) [inline site/team/triathlon/spartenleitung_triathlon/index.html:12]: Decide if this should be a list or all other pages should not be a list

File: site/team/triathlon/trainer_triathlon/index.html [hunk 17]
```diff
@@ -0,0 +1,24 @@
...
+<p><strong>Funktion: </strong>Schwimmtrainer Anf&auml;nger</p>
+<p><strong>Name: </strong>Michael Bacher</p>
+
+<p>Ich bin 40 Jahre alt und hab vor ca. 6 Jahren mit dem Triathlon begonnen, da meine Lieblingsdiziplin das Schwimmen ist, m&ouml;chte ich diese Leidenschaft euch gerne weitergeben.</p>
+<p><strong>Mein Alter</strong>: AK 40</p>
+<p><strong>Meine liebste Sportart</strong>: Triathlon</p>
+<p><strong>Meine liebste Trainingsform</strong>: Schwimmen und Rad fahren</p>
+<p><strong>Mein h&auml;rtester Wettkampf</strong>: Allg&auml;u Triathlon 2019</p>
+<p><strong>Mein verr&uuml;cktester Wettkampf</strong>: -</p>
+<p><strong>Meine besten Leistungen</strong>: 1. Marthon Berlin</p>
```
Comment (TomasKostadinov) [inline site/team/triathlon/trainer_triathlon/index.html]: Same here

**Review by TomasKostadinov — CHANGES_REQUESTED** [review]
Some things still need to be changed, but this is a very big part of the site and already looks very good!

File: site/team/triathlon/index.html (hunk 15 above)
Comment (Urs-sein-github) [reply site/team/triathlon/index.html]: No, because i don't know how.

File: site/team/presse/index.html (hunk 14 above)
Comment (Urs-sein-github) [reply site/team/presse/index.html]: This is my purpose, I won't betray my god.

File: site/team/leichtathletik/spartenleitung_leichtathletik/index.html (hunk 9 above)
Comment (Urs-sein-github) [reply site/team/leichtathletik/spartenleitung_leichtathletik/index.html:7]: was meant as a spaceholder. Shall it still be removed?

Comment (TomasKostadinov) [reply site/team/leichtathletik/spartenleitung_leichtathletik/index.html:7]: Yes please

File: site/team/leichtathletik/spartenleitung_leichtathletik/index.html [hunk 18]
```diff
@@ -0,0 +1,26 @@
...
+    <li><strong>Alter</strong>: 48 Jahre</li>
+    <li><strong>Meine Lieblingssportart</strong>: Leichtathletik Mehrkampf</li>
+    <li><strong>Mein H&auml;rtester Wettkampf</strong>: erster Zehnkampf als Jugendlicher</li>
+    <li><strong>Meine besten Leistungen</strong>:
+    <ul>
+    <li>zweimal Deutscher Meister</li>
+    <li>zweimal w&uuml;rttembergischer Meister</li>
+    <li>Leistungskader im Weitsprung und Sprint</li>
+    </ul>
+    </li>
```
Comment (TomasKostadinov) [inline site/team/leichtathletik/spartenleitung_leichtathletik/index.html]: Closing tag but no opening tag?

File: site/team/leichtathletik/trainer_leichtathletik/index.html [hunk 19]
```diff
@@ -0,0 +1,9 @@
...
+<div class="row">
+    <div class="container">
+    </br>
```
Comment (TomasKostadinov) [inline site/team/leichtathletik/trainer_leichtathletik/index.html]: Remove /br and change `container` to  `container distance`

File: site/team/organisation_silvesterlauf/julia_schier/index.html [hunk 20]
```diff
@@ -0,0 +1,24 @@
+<div class="container distance">
+  
+    
+    </br>
+        <h2>Hey, ich bin die Julia Schierk</h2>
+        
+        <img src="">
```
Comment (TomasKostadinov) [inline site/team/organisation_silvesterlauf/julia_schier/index.html]: No image tag needed if there is no image

File: site/team/triathlon/spartenleitung_triathlon/index.html [hunk 21]
```diff
@@ -0,0 +1,17 @@
+<div class="container distance">
+</br>
```
Comment (TomasKostadinov) [inline site/team/triathlon/spartenleitung_triathlon/index.html]: /br is closing a br tag. We did not open a br tag so no need of this

**Review by TomasKostadinov — CHANGES_REQUESTED** [review]
Small things left. Also some things from the last review are not fixed

File: site/team/presse/index.html [hunk 22]
```diff
@@ -1,4 +1,4 @@
-<meta http-equiv="refresh" content="5; url=https://www.fetzer-bau.de/" />
+<meta http-equiv="refresh" content="10; url=https://www.kein-taeter-werden.de/" />
```
Comment (DominikRuf) [inline site/team/presse/index.html]: Please remove it, that we can finally merge this content to master.

**Comment by BlueEnni** [comment]
@Urs-sein-github maybe you can compress your images as well - the site looks great otherwise but the image loading time is to extreme

**Comment by TomasKostadinov** [comment]
<p align="center">
API Giphy logo
</p>

**Review by TomasKostadinov — APPROVED** [review]

**Pull request merged**